package.json

```diff
@@ -22,7 +22,7 @@
     "!protractor.conf.js"
   ],
   "scripts": {
-    "build": "yoshi build && npm run copy-styleable-ext && npm run compile-base-ui && npm run generate-stylable-index && npm run storybook:build",
+    "build": "npm run create-icons && yoshi build && npm run copy-styleable-ext && npm run compile-base-ui && npm run generate-stylable-index && npm run storybook:build",
```

Comment:
if it's a yoshi4 project you can just import svgs as a component (using https://github.com/smooth-code/svgr)

Reply:
it's not 😞